nixos/modules/system/boot/systemd/tmpfiles.nix

```diff
@@ -100,6 +155,18 @@ in
           ${concatStringsSep "\n" cfg.rules}
         '';
       })
+      (pkgs.runCommand "systemd-extra-tmpfiles" {} ''
+        mkdir -p $out/lib/tmpfiles.d
+        cd $out/lib/tmpfiles.d
+
+        ${concatMapStrings (namespace:
+          concatMapStrings (path:
+            concatMapStrings (entry: ''
+              echo "${entry.type} ${path} ${entry.mode} ${entry.user} ${entry.group} ${entry.age} ${entry.argument}" >> nixos-${namespace}.conf
```

Comment:
This should probably be `10-nixos-${namespace}.conf` to make sure it runs first?

or `99` for last. we need to make some decision here.

Reply:
both seem fine for me? we have also a bunch of default ones in there which don't have a prefix number

Reply:
since we're using attribute sets we should really consider putting our options higher priority - ready to override because of module merging, more power

Reply:
We should honestly leave this to the user, so `"${namespace}.conf"`. The the user can decide for themselves whether they need priority over other settings or not. This is also in line with what we do with many other systemd tools in Nixpkgs.